lightning/src/ln/onion_utils.rs

```diff
@@ -1010,10 +1084,10 @@ fn decode_next_hop<T, R: ReadableArgs<T>, N: NextPacketBytes>(shared_secret: [u8
 		Err(err) => {
 			let error_code = match err {
 				msgs::DecodeError::UnknownVersion => 0x4000 | 1, // unknown realm byte
-				msgs::DecodeError::UnknownRequiredFeature|
-				msgs::DecodeError::InvalidValue|
-				msgs::DecodeError::ShortRead => 0x4000 | 22, // invalid_onion_payload
-				_ => 0x2000 | 2, // Should never happen
+				msgs::DecodeError::UnknownRequiredFeature
+				| msgs::DecodeError::InvalidValue
+				| msgs::DecodeError::ShortRead => 0x4000 | 22, // invalid_onion_payload
+				_ => 0x2000 | 2,                                 // Should never happen
```

Comment:
Confused why it added all this indentation before the comment

Reply:
Honestly, I'd prefer to just move the comment above each line rather than having them at the end of each line.